src/ImageRunModal.jsx

```diff
@@ -635,6 +638,72 @@ export class ImageRunModal extends React.Component {
         return owner === systemOwner;
     };
 
+    prefillModal() {
+        const container = this.props.container;
+        const containerDetail = this.props.containerDetail;
+        const image = this.props.localImages.find(img => img.Id === container.ImageID);
+
+        if (containerDetail.Config.CreateCommand) {
+            this.setState({
+                dialogWarning: _("This container was not created by cockpit"),
+                dialogWarningDetail: _("Some options may not be copied to the new container."),
+            });
```

Comment:
Isn't this something we need to always show? We won't support all options podman has as we won't ever have a feature complete UI.

Reply:
I think it's okay to not show it when the container is created through cockpit. On the other hand the "check" if it is made by cockpit isn't 100% robust as it only checks whether it was created through commandline or podman API.